src/video/SDL_video_capture.m

```diff
@@ -30,6 +30,9 @@
 #include "SDL_video_capture_c.h"
 #include "../thread/SDL_systhread.h"
 
+#ifdef __MACOSX__
+#  include <AvailabilityMacros.h>
```

Comment:
You need to `#include <AppKit/NSApplication.h>` ,  I think.

PS: Someone should still change cmake side properly (not me), and review the thing.

Reply:
I was about to change to this :)